english/beach/beach.md

```diff
@@ -0,0 +1,5 @@
+You know in your heart the beach is where you wanna be. The sun, the sand, the sea and the breeze. You close your eyes and you can just imagine the beauty. Without any hesitation you head for the door.
+
+Do you:
+[Call your friends?](https://github.com/jonotko/create-your-own-adventure/blob/jono-adventure/english/go-with-friends/go-with-friends.md)
+[Proceed to go alone?](https://github.com/jonotko/create-your-own-adventure/blob/jono-adventure/english/go-alone/go-alone.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.